docs/src/gauss_newton_kalman_inversion.md

```diff
@@ -0,0 +1,57 @@
+# [Gauss Newton Kalman Inversion](@id gnki)
+
+### What Is It and What Does It Do?
+Gauss Netwon Kalman Inversion (GNKI) ([Chada et al, 2020](https://arxiv.org/pdf/2010.13299)), also known as the Iterative Ensemble Kalman Filter with Statistical Linearization, is a derivative-free ensemble optimizaton method based on the Gauss Newton optimization update and the Iterative Extended Kalman Filter (IExKF) ([Jazwinski, 2007]).  In the linear case and continuous limit, GNKI recovers the true posterior mean and covariance.  Empirically, GNKI performs well as an optimization algorithm in the nonlinear case.  
```

Comment:
Please use DOI links over regular URLS. 
 Jazwinski needs a link too

Reply:
I replaced the regular URL with the DOI link.
The Jazwinski reference is a book and there is no open source ebook.  I included a google books link for now (it is what comes up on google scholar for the book), but I'm not sure if it would be better to not include any link?